recipes-examples/celebrity-face-match/demo-celebrity-face-match/demo-celebrity-face-match.service

```diff
@@ -1,16 +1,14 @@
 [Unit]
 Description=Celebrity Face Match Demo Service
-Wants=systemd-udev-settle.service weston.service
-After=systemd-udev-settle.service weston.service
+Requires=weston.service
```

Comment:
Any reason why you remove the `systemd-udev-settle.service`?
If I remember correctly, I added these to make sure the device links like `/dev/cam-csi1` and `/dev/video-isi-csi1` are created before the demo starts.

Reply:
I never experienced issues without the dependency to `systemd-udev-settle.service`, although this may be just luck in timing. Will add it back.